blocks/table/table.css

```diff
@@ -72,6 +72,20 @@ main .table.stats td {
   text-align: center;
 }
 
+main .table.weather tr {
+  background-color: transparent !important;
```

Comment:
you may be able to avoid the `!important` here by moving this declaration down below the general table background styles on line 107-113